src/app/components/event/steps/Preview.vue

```diff
@@ -0,0 +1,255 @@
+              >
+                <img
+                  :src="props.form.previewUrls[1]"
+                  class="h-full w-full object-cover"
+                  alt="Second highlight"
+                />
+              </div>
+
+              <div class="relative aspect-[4/3]">
+                <div
+                  v-if="props.form.previewUrls.length > 2"
+                  class="relative h-full"
+                >
+                  <img
+                    :src="props.form.previewUrls[2]"
+                    class="h-full w-full object-cover"
+                    alt="Third highlight"
+                  />
+                  <div
+                    v-if="props.form.previewUrls.length > 3"
+                    class="group absolute inset-0 flex cursor-pointer items-center justify-center bg-black/50"
+                  >
+                    <div class="text-center text-white">
+                      <div class="mb-1 flex items-center justify-center">
+                        <Grid class="h-6 w-6" />
+                      </div>
+                      <span class="text-sm font-medium">{{ t('seeAll') }}</span>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div
+          v-else
+          class="rounded-2xl border-2 border-dashed border-gray-300 p-12 text-center"
+        >
+          <div class="flex flex-col items-center">
+            <Image class="mb-4 h-12 w-12 text-gray-400" />
+            <p class="text-gray-500">{{ t('noHighlights') }}</p>
+          </div>
+        </div>
+
+        <div class="mt-6">
+          <h2 class="mb-3 text-2xl font-bold">{{ t('organizer') }}</h2>
+          <div class="flex items-center gap-4">
+            <AccountProfilePicture
+              :account-id="store.signedInAccountId || ''"
+              classes="h-16 w-16 rounded-full bg-gray-100"
+              height="64"
+              width="64"
+            />
+            <h3 class="text-xl font-semibold text-[#5B47D9]">
+              {{ store.signedInUsername }}
+            </h3>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue'
+import { Share, Heart, MoreHorizontal, Grid, Image } from 'lucide-vue-next'
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+} from '@/components/scn/carousel'
+
+const { t } = useI18n()
+const currentSlide = ref(0)
+const store = useMaevsiStore()
+
+const handleSlideChange = (index: number) => {
+  currentSlide.value = index
+}
+
+const props = defineProps({
+  form: {
+    type: Object,
+    required: true,
+  },
+  validation: {
+    type: Object,
+    required: true,
+  },
+})
+
+const formatDate = (
+  dateString: string,
+  options: Intl.DateTimeFormatOptions,
+) => {
+  const date = new Date(dateString)
+  return new Intl.DateTimeFormat('de', options).format(date)
+}
```

Comment:
This looks like a good candidate to extract into utils or a composable as many parts of the code will likely use this.

Reply:
Applies to `pages/event/preview/index.vue` too